_beats/dev-tools/common.bash

```diff
@@ -39,12 +39,14 @@ get_go_version() {
 # To read more about installing gvm in other platforms: https://github.com/andrewkroh/gvm#installation
 install_gvm() {
   # Install gvm
-  if [ ! -f "/usr/local/bin/gvm" ]; then
-    curl -sL -o ~/bin/gvm https://github.com/andrewkroh/gvm/releases/download/v0.3.0/gvm-linux-amd64
-    chmod +x /usr/local/bin/gvm
+  GVMDIR="$HOME/.gvm"
+  GVM="$GVMDIR/gvm"
```

Comment:
If the pipeline makes sure that GVMDIR is in the PATH, then LGTM :)

Reply:
Oh right, it doesn't. I guess we should just add it to $PATH here?